src/preload/index.js

```diff
@@ -0,0 +1,10 @@
+import { contextBridge, ipcRenderer } from "electron";
+
```

Comment:
The notifications still don't seem to work with this, but I was also trying the same thing, so... I'll just keep prodding this